mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,366 @@
+"""Test full 1D diode run with diagnostics."""
+# [[[TODO]]] NOT TOUCHED, JUST COPIED FROM WARP, SO FAR
+from __future__ import division
+
+from builtins import range
+# import os
+# import sys
+
+import pytest
+import numpy as np
+
+from pywarpx import picmi
+# import pywarpx
+# import mewarpx
+
+# Necessary to load Warp as library (because Warp processes arguments even when
+# loaded as a module!!!)
+# del sys.argv[1:]
+
+# import warp
+# from warp.data_dumping.openpmd_diag import ParticleDiagnostic
+
+# from metools import analysis, diags
+# from metools import init_restart_util, runtools, util, warputil
+from mewarpx.setups_store import diode_setup
+
+
+constants = picmi.constants
+
+##########################
```

Comment:
All parameters should be inside the test function - only imports and global variables across all tests belong outside. Hopefully most of these can be directly passed to the DiodeRun setup rather than being declared separately.

Reply:
Some of the parameters make it clearer how the values of certain things like DT are obtained. For example `DT = 1.0 / (400 * FREQ)` is better than `DT = 1.0 / (400 * 13.56e6)`. `N_INERT` and `T_INERT` are used to create the uniform distributions so they seem to make sense as variables as well to me.